backend/LexBoxApi/Services/HgService.cs

```diff
@@ -257,6 +259,7 @@ private static void CopyFilesRecursively(DirectoryInfo source, DirectoryInfo tar
     private static void SetPermissionsRecursively(DirectoryInfo rootDir)
     {
         if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) return;
+        rootDir.UnixFileMode = Permissions;
```

Comment:
We never explicitly set the permissions of `target` in `CopyFilesRecursively`. Should we add that too?

Reply:
yes. Done